pom.xml

```diff
@@ -167,7 +167,7 @@
     <dependency>
       <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-etl-proto</artifactId>
-      <version>${cdap.version}</version>
+      <version>6.8.0-SNAPSHOT</version>
```

Comment:
update the value for CDAP version, don't remove this reference

Reply:
Done